README.md

```diff
@@ -1,17 +1,57 @@
-## Introduction
+This project contains a test suite which uses the [SpatialOS GDK for Unreal](https://github.com/spatialos/UnrealGDK). 
 
-You can use the SpatialOS Unreal GDK Test Suite for testing and validating your development with the SpatialOS Unreal GDK.
+It is used to validate the functionality of the SpatialOS GDK. The test suite has tests for several key components of the GDK such as ctor replication and the execution of RPCs.  
 
-For more information, see the [SpatialOS Unreal GDK repository](https://github.com/improbable/UnrealGDK) and its [documentation](https://github.com/improbable/UnrealGDK/blob/master/docs/readme.md).
+For setup instructions, follow the [Get started](https://docs.improbable.io/unreal/latest/setup-and-installing) guide in the GDK for Unreal documentation. This guide uses the [GDK for Unreal Starter Project](https://github.com/spatialos/UnrealGDKStarterProject) as an example. When you follow the installation instructions, you need to replace the "Starter Project" repository with the [Test Suite](https://github.com/spatialos/UnrealGDKTestSuite) one.
 
-## Helper scripts
+For more information, see the [SpatialOS GDK for Unreal documentation](https://docs.improbable.io/unreal/latest/).
+
+####  Helper scripts
 
 This repository contains a set of helper scripts to get you started.
 
 | Helper script | Description |
 | --- | --- |
-| LaunchSpatial.bat | Starts a local Spatial deployment with the default launch configuration |
-| LaunchServer.bat | Starts an Unreal server worker, and connects it to the local deployment |
-| LaunchClient.bat | Starts an Unreal client worker, and connects it to the local deployment |
\ No newline at end of file
+| `LaunchSpatial.bat` | Starts a local SpatialOS deployment with the default launch configuration. |
+| `LaunchServer.bat` | Starts an Unreal server-worker, and connects it to the local deployment. |
+| `LaunchClient.bat` | Starts an Unreal client-worker, and connects it to the local deployment. |
+| `ProjectPaths.bat` | Used by the `LaunchClient.bat`, `LaunchServer.bat` and `LaunchSpatial.bat` to specify the project environment when those scripts are run. |
+
+#### Game controls
+
+The following controls are bound in-game, for gameplay and testing.
+
+| Key binding | Function |
+| --- | --- |
+| W,A,S,D | Standard movement |
+| Space | Jump |
+| Shift | Sprint |
+| K | Trigger test suite execution |
+
+#### Give us feedback
+We have released the GDK for Unreal this early in development because we want your feedback. Please come and talk to us about the software and the documentation via: [Discord](https://discordapp.com/channels/311273633307951114/339471548647866368) - [Forums](https://forums.improbable.io/) - [GitHub issues in this repository](https://github.com/spatialos/UnrealGDK/issues).
+
+#### Where to get the GDK and example projects
+The GDK and its starter projects are available on GitHub.
+* [GDK: github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
```

Comment:
There is usually a warning about the 404 if you aren't logged in as an Unreal user (just was reminded of this).